packages/graphback-datasync/src/util.ts

```diff
@@ -1,6 +1,15 @@
-import { ModelDefinition } from '@graphback/core';
-import { parseMetadata } from 'graphql-metadata'
+import { ModelDefinition, GraphbackCRUDService } from '@graphback/core';
+import { parseMetadata } from 'graphql-metadata';
+import { DataSyncCRUDService } from "./services";
 
 export function isDataSyncModel(model: ModelDefinition): boolean {
   return parseMetadata("delta", model.graphqlType) && parseMetadata("versioned", model.graphqlType)
 }
+
+export function getDataSyncService(service: GraphbackCRUDService): DataSyncCRUDService {
+  if (service instanceof DataSyncCRUDService) {
+    return service;
+  }
+
+  throw Error("Service is not a DataSyncCRUDService. Please use DataSyncCRUDService and DataSync-compliant DataProvider with DataSync Plugin to get Delta Queries.")
```

Comment:
This error is really only useful in the `DataSyncPlugin`, but if someone wants to fetch the DataSyncCRUDService or undefined (quite common) they then have to catch an error. I'd throw the error in the DataSyncPlugin instead and change this to `isDataSyncService`